CODE_OF_CONDUCT.md

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Not sure if this change is intended?

Reply:
Those are related to prettier formatting.

Here is the websocket commit: https://github.com/robbie-cahill/tunnelmole-service/pull/6/commits/580c8bed33ee05f90ca418aa1e972101aa091876

Reply:
i'll try and get this commit running locally, if it works we might be able to cherry pick it